templates/alcatel_sros_ping.textfsm

```diff
@@ -0,0 +1,14 @@
+Value SENT_QTY (\d+)
+Value DESTINATION (\d+.\d+.\d+.\d+)
+Value LOSS_PCT (\d+)
+Value SUCCESS_QTY (\d+)
```

Comment:
Same as your other PR. Make sure the values follow the existing Value names and then we required tests.

Please reference the README for more details or join our Slack for help.